### ISSUE TITLE
Error: Database not found

### ISSUE THREAD

**Issue by carvalhar** [pjheslin/diogenes#11]
It's there, it worked before, but now it doesn't.

**Comment by carvalhar** [comment]
![Capture](https://user-images.githubusercontent.com/20246049/57589071-de2f6100-74f4-11e9-8691-542d48377383.PNG)

**Comment by carvalhar** [comment]
Solved it by copying from an external HD to the root drive. 

I think you have a problem loading from external HDs or from a long path.

**Comment by carvalhar** [comment]
Maybe because the path has a latin diacritic "á"?

**Comment by pjheslin** [comment]
Yes, I think it is because there is an accented character in the path

**Comment by pjheslin** [comment]
I have tried to fix this bug, but I've given up for now.  In the current code on my own windows machine, with non-ascii paths, searching works, and browsing partially works but fails in the end.  It's a very strange bug, and I have reached an impasse with it.

In any case, I'm not confident that my Windows code will work in all circumstances.  So I am leaving this to one side for now.  I will return to it when I have time to understand Windows non-ascii filepath handling in general.

I've added this to the to-do list and I will update the FAQ to indicate that Windows users should not have non-ascii characters in the paths to the databases.
